client/python/lib/test/test_tensors.py

```diff
@@ -104,3 +104,88 @@ def test_make_ndarray_invalid_no_type():
 
     exception = exception_info.value
     assert str(exception) == "Tensor data type invalid"
+
+def test_make_tensor_proto_valid_int():
+    values = [1, 2, 3]
+    tensor_proto = make_tensor_proto(shape=[3], dtype = DataType.DT_INT8, values = values)
+    np_values = np.array(values, np.int8)
+    assert tensor_proto.tensor_content == np_values.tobytes()
+    assert tensor_proto.dtype == DataType.DT_INT8
+
+def test_make_tensor_proto_valid_float():
+    values = [1.0, 2.0, 3.0]
+    tensor_proto = make_tensor_proto(shape=[3], dtype = DataType.DT_FLOAT, values = values)
+    np_values = np.array(values, np.float32)
+    assert tensor_proto.tensor_content == np_values.tobytes()
+    assert tensor_proto.dtype == DataType.DT_FLOAT
+
+def test_make_tensor_proto_valid_scalar():
+    values = 5.0
+    tensor_proto = make_tensor_proto(shape=[3], dtype = DataType.DT_FLOAT, values = values)
+    np_values = np.array(values, np.float32)
+    assert tensor_proto.tensor_content == np_values.tobytes()
+    assert tensor_proto.dtype == DataType.DT_FLOAT
+
+def test_make_tensor_proto_valid_string():
+    values = bytes([0x13, 0x00, 0x00, 0x00, 0x08, 0x00])
+    tensor_proto = make_tensor_proto(shape=[4], dtype = DataType.DT_STRING, values = [values])
```

Comment:
Why passing shape [4] if batch=1? Shape should be [1]. I think we should also validate if number binary images passed to make_tensor_proto matches the shape? @michalkulakowski @dtrawins @mkuczyns11

Reply:
The same goes for non binary data, we specify shape but we should also validate data if is in same shape as specified in parameter

Reply:
I agree. With that config:
`data = 5.0`
`tensor_proto = make_tensor_proto(data, shape = [3])`
In output tensor_shape is 
`dim {
  size: 3
}`, but the data provided has is a scalar.